src/dom-wrapper-error.ts

```diff
@@ -0,0 +1,58 @@
+import { FindComponentSelector, DOMWrapperAPI } from './types'
+
+interface Options {
+  selector: FindComponentSelector
+}
+
+export class DOMWrapperError implements DOMWrapperAPI {
+  private selector: FindComponentSelector
```

Comment:
Do we need this or can it be removed? I think we would be better off, as it doesn't look used? Same for the constructor below, and we could get rid of the `Options` interface

Reply:
I don't think this is used. I will look into this, be good to remove it.